.github/workflows/ci.yml

```diff
@@ -405,6 +404,24 @@ jobs:
       - name: Run the tests
         run: invoke ci-tests -m unit --xdist --slowest=10  --timeout=2.0
 
+  pydantic-v1:
```

Comment:
Added a test for our min version of pydantic now that the other tests will be installing v2